stratum/hal/lib/common/yang_parse_tree_test.cc

```diff
@@ -109,15 +109,17 @@ class YangParseTreeTest : public ::testing::Test {
   }
 
   // A proxy for YangParseTree::AddSubtreeInterface().
-  void AddSubtreeInterface(const std::string& name) {
+  void AddSubtreeInterface(const std::string& name,
+                           SingletonPort* singleton_p) {
```

Comment:
You can use `SingletimePort* singleton_p = nullptr` for default value, so you don't need to modify line 176, 575, 707, ... etc,